lib/package.json

```diff
@@ -72,4 +79,5 @@
         "react": ">=16.8",
         "react-dom": ">=16.8"
     }
+
```

Comment:
```suggestion
```

Reply:
done with the changes @pavinduLakshan

Reply:
Thanks @s-vamshi , please update the PR title as well to accurately describe the changes.

Reply:
hey @pavinduLakshan updated pr title